File: Sorting/bubbleSort.java
public class bubbleSort {

    public void bsort(int a[])
    {
        int n=a.length;
        for(int i=0;i<n;i++)
        {
            for(int j=0;j<n-i-1;j++)
            {
                if(a[j]>a[j+1])
                {
                    a[j+1]=a[j]+a[j+1];
                    a[j]=a[j+1]-a[j];
                    a[j+1]=a[j+1]-a[j];
                }
            }
        }
        
    }
    public static void main(String args[])
    {
        int a[]={1,5,4,6,3,7,2};
        bubbleSort bs = new bubbleSort();
        bs.bsort(a);

        for(int i=0;i<a.length;i++)
        {
            System.out.print(a[i]+" ");
        }
    }
}

File: Sorting/bubbleSort.py
def bubbleSort(a):
    n=len(a)
    for i in range(0,n):
        for j in range(0,n-i-1):
            if( a[j] > a[j+1] ):
                temp = a[j]
                a[j] = a[j+1]
                a[j+1] = temp
a=[1,44,6,4,3,9,8,0]
bubbleSort(a)
print(a)